wagtail_wordpress_import/importers/import_hooks.py

```diff
@@ -19,12 +19,14 @@ def __init__(self):
             setattr(self, hook, [])
 
     def process(self, imported_pages):
-        """Run all hooks in the config for each page
-        The function defined in the config key FUNCTION will be imported
-        and run on each page"""
+        """Run all hooks in the config for each page.
+
+        Each function defined in the config key "FUNCTION" will be run on each page.
+        """
+        registry = self._get_hook_handler_data()
```

Comment:
**Comment**: We only need to call the `._get_hook_handler_data()` setup method once per import. Its return value doesn't change.